### PR TITLE
Cross referenced data exploration and ml feasibility study, added data quality assessment to data exploration objectives

### PR DESCRIPTION
**Pull request by sarahdettorre**
Cross referenced data exploration and ml feasibility study, added data quality assessment to data exploration objectives

# Pull Request Template

## What are you trying to address

- Clarify relationship between feasibility study and data exploration
- Highlight data quality assessment as a data exploration objective

## Description of new changes

- Cross referencing feasibility study and data exploration for clarity on how they are related to one-another
- Added brief point on data quality analysis in data exploration stage

## For all pull requests

- [ ] Link to the issue you are solving (so it gets closed)
- [ ] Label the pull request with the appropriate area(s)
- [ ] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [ ] Assign the project - Engineering Playbook Backlog
- [ ] Assign the pull request to yourself

### CODE REVIEW

File: docs/machine-learning/ml-feasibility-study.md [hunk 1]
```diff
@@ -37,8 +37,9 @@ Collaboration from individuals with diverse skill sets is desired at this stage,
 
 ### Data discovery
 
-* Hold a data exploration workshop and deep dive with domain experts
+* Hold a [data exploration](./ml-data-exloration.md) workshop and deep dive with domain experts
 * Understand the data dictionary, data quality and access (availability)
+* Understand the quality of the data. Is there already a data validation strategy in place?
```
Comment (omri374) [inline docs/machine-learning/ml-feasibility-study.md:42]: Data quality is already mentioned in the previous bullet. Consider removing it from there.

**Review by omri374 — APPROVED** [review]
Looks great! One tiny comment

File: docs/machine-learning/ml-data-exploration.md [hunk 2]
```diff
@@ -48,8 +48,9 @@ Key objectives of the exploration workshops include the following:
...
 2. What order of magnitude is the current data (e.g., GB, TB)? Is this all relevant?
 3. How does the organization decide when to collect additional data or purchase external data? Are there any examples of this?
-4. What data has been used so far to analyse recent data-driven projects? What has been found to be most useful? What was not useful? How was this judged?
-5. What additional internal data may provide insights useful for data-driven decision-making for proposed projects? What external data could be useful?
-6. What are the possible constraints or challenges in accessing or incorporating this data?
-7. How was the data collected? Are there any obvious biases due to how the data was collected?
-8. What changes to data collection, coding, integration, etc has occurred in the last 2 years that may impact the interpretation or availability of the collected data
+4. Understand the quality of the data. Is there already a data validation strategy in place?
+5. What data has been used so far to analyse recent data-driven projects? What has been found to be most useful? What was not useful? How was this judged?
```
Comment (shiranr) [inline docs/machine-learning/ml-data-exploration.md:52]: NIT: Perhaps split this up:
5. Previous data usage:
a. Which data used ... ? 
b. What has been found  ... ?

>

**Review by shiranr — APPROVED** [review]
LGTM, small comment

File: docs/machine-learning/ml-feasibility-study.md (hunk 1 above)
Comment (sarahdettorre) [reply docs/machine-learning/ml-feasibility-study.md:43]: Good catch! I split up data discovery bullet point on availability, dictionary and quality.

**Pull request merged**